### PR TITLE
Updated wgs alignment qc to use version 4.2.4 (prod)

### PR DESCRIPTION
**Pull request by alexiswl**
Updated pipeline ID (from wgs qc to alignment pipeline and version from 3.9.3 to 4.2.4)

Allow for WTS QC to use this template

### CODE REVIEW

**Comment by victorskl** [comment]
[Portal Release - Ocicat: Run WTS QC workflow through Portal before running Dragen WTS Tumor Only Workflow](https://trello.com/c/H0DQyo2B/1485-portal-release-ocicat-run-wts-qc-workflow-through-portal-before-running-dragen-wts-tumor-only-workflow)

**Comment by victorskl** [comment]
Checklist

_before pushing to Portal Production _

* [ ] #341 must be merged and did run new QC pipeline automation in Staging, first

**Review by reisingerf — APPROVED** [review]

**Comment by victorskl** [comment]
I will be rebasing this onto latest master HEAD. Hence, force push to update the branch. And resolve the conflict.

**Review by victorskl — APPROVED** [review]
Assert that

File exists
```
ica files get gds://production/reference-data/dragen_wts/hg38/gencode.v39.annotation.gtf
```

File exists
```
ica files get gds://production/reference-data/dragen_hash_tables/v9-r3/hg38-alt_masked-cnv-hla-rna/hg38-alt_masked.cnv.hla.rna-9-r3.0-1.tar.gz
```

Workflow version exists and equal to Staging 
```
ica workflows versions list wfl.c07ecc424a5a41bd94abe9ef519867a0 | grep 4.2.4--5bfabd0
```

Validation Run in Staging
- https://umccr.slack.com/archives/C017F7HHWF6/p1699345245594809

File: terraform/stacks/umccr_data_portal/workflow/wgs_alignment_qc.tf [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    prod = "wfl.23f61cb1baab412a8c37dc93bed6c2af"
+    prod = "wfl.c07ecc424a5a41bd94abe9ef519867a0"
     stg  = "wfl.c07ecc424a5a41bd94abe9ef519867a0"
   }
 
   wgs_alignment_qc_wfl_version = {
     dev  = "4.2.4"
-    prod = "3.9.3--4e00721"
+    prod = "4.2.4--5bfabd0"
     stg  = "4.2.4--5bfabd0"
```
Comment (reisingerf) [inline terraform/stacks/umccr_data_portal/workflow/wgs_alignment_qc.tf:11]: Thanks!
That's much less confusing now!

**Review by reisingerf — APPROVED** [review]

**Comment by reisingerf** [comment]
(Existence of workflow version, reference files in production verified)

**Review by ohofmann — COMMENTED** [review]
Go :)

**Comment by victorskl** [comment]
Merging...

**Pull request merged**